modules/testcase.py

```diff
@@ -19,6 +19,7 @@ def test_create_room_successfully(self):
         initial_room_count = len(self.dojo.all_rooms)
         blue_office = self.dojo.create_room("Blue", "Office")
         new_livingSpace = self.dojo.create_room("St. Catherines", "livingSpace")
+        re_add_livingSpace = self.dojo.create_room("St. Catherines", "livingSpace")
         self.assertTrue('blue_office')
```

Comment:
1. assert successful return statements such as 'room created successfully' 
2. You can have multiple asserts but its good practice to have one main assert statement so one can know what exactly you are testing for in a single test.
3. add docstrings to your test functions to describe what you are testing for.